app/models/user.rb

```diff
@@ -193,6 +193,11 @@ def enable_mfa!(seed, level)
     save!(validate: false)
   end
 
+  def mfa_write_authorized?(otp)
+    return true unless mfa_login_and_write?
+    otp_verified?(otp || '')
```

Comment:
please don't do `otp || ''`, either add default argument to `mfa_write_authorized?` or `otp_verified?`.

Reply:
I think I can replace `otp || ''` with `otp.to_s`